src/commands/base.js

```diff
@@ -420,6 +429,7 @@ class Command {
 	isUsable(message = null) {
 		if(!message) return this._globalEnabled;
 		if(this.guildOnly && message && !message.guild) return false;
+		if(this.dmOnly && message && message.channel.type !== 'dm') return false;
```

Comment:
```suggestion
		if(this.guildOnly && !message.guild) return false;
		if(this.dmOnly && message.channel.type !== 'dm') return false;
```
looks good to me aside from this, checking the truthiness of `message` isnt needed seeing as its done above already

Reply:
Sorry, might have to explain this to me xd. My understanding was that checking the message for truthiness is to check it isnt undefined? If this is correct and the message was undefined it wouldn't run through the first if statement and would check message.channel.type and throw an error of cant access property channel or type of undefined?

Reply:
Well, yeah, but a check is already done for it above, so checking again is useless.